streams/piping/close-propagation-forward.any.js

```diff
@@ -166,7 +163,7 @@ promise_test(() => {
 
   const pipePromise = rs.pipeTo(ws);
 
-  setTimeout(() => rs.controller.close());
+  step_timeout(() => rs.controller.close());
```

Comment:
Technically since this is not a single-page test, you should use t.step_timeout(). See https://web-platform-tests.org/writing-tests/testharness-api.html#timeouts-in-tests.

However, it doesn't seem to make any actual difference.

Reply:
If I understand correctly, the `step_*` functions ensure that any `assert_*` calls are scoped to the correct test. However, for these tests, we don't do any assertions inside these timeout callbacks, so it doesn't really matter.

For good measure, I'll update the tests to use `t.step_timeout` anyway. There's one usage in `resources/rs-utils.js` where there's no obvious reference to a `Test` object, so there I'll keep the global `step_timeout`.